### PR TITLE
Add a doc for numeric IDs

### PR DESCRIPTION
**Pull request by iwahbe**
Fixes https://github.com/pulumi/pulumi-terraform-bridge/issues/352

### CODE REVIEW

File: pf/internal/check/checks.go [hunk 1]
```diff
@@ -78,26 +75,79 @@ func checkIDProperties(sink diag.Sink, info tfbridge.ProviderInfo, isPFResource
 	return nil
 }
 
-func resourceHasRegularID(resource shim.Resource, resourceInfo *tfbridge.ResourceInfo) (bool, string) {
+type resourceError struct {
```
Comment (VenelinMartinov) [inline pf/internal/check/checks.go:78]: These don't look like docs changes - is this intentional?

Comment (iwahbe) [reply pf/internal/check/checks.go:78]: It's not a docs change and it is intentional. I can move the change to a separate PR.

File: docs/resource-ids.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+error: Resource dnsimple_email_forward has a problem: "id" attribute is of type "int", expected type "string". To map this resource consider overriding the SchemaInfo.Type field or specifying ResourceInfo.ComputeID.
+```
+
+This error happens when the upstream resource has an ID but it's not a string. You can fix
+it by setting the `SchemaInfo.Type` override for the `"id"` field:
+
+```go
+			"dnsimple_email_forward": {
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"id": {Type: "string"},
```
Comment (t0yv0) [inline docs/resource-ids.md:29]: This only works if the actual type can be coalesced to string right? Like it's a number. It is not an object.

File: docs/resource-ids.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+require an ID field of type string set on the provider. If the provider being bridged is
+based on SDKv1 or SDKv2, then the bridge handles Pulumi's ID field without intervention.
+
+## PF Based Providers
+
+Most PF based providers have an attribute called `"id"` of the right kind for the bridge
+to use. If your provider doesn't, then you will see an error when `make tfgen` is
+run. Each error has a different kind of resolution.
+
+### ID of the wrong type
```
Comment (t0yv0) [inline docs/resource-ids.md:18]: Section on missing "id" fields? I was recently populating ComputeID in aws again. Did we deprecate that or that's still the way?

**Review by t0yv0 — APPROVED** [review]

File: docs/resource-ids.md (hunk 2 above)
Comment (iwahbe) [reply docs/resource-ids.md:29]: Yes. I'll add a note explaining that.

**Pull request merged**